example/build.gradle

```diff
@@ -9,8 +9,8 @@ android {
         applicationId "id.vouched.android.java.example"
         minSdkVersion 22
         targetSdkVersion 30
-        versionCode 10
-        versionName "1.0.1"
+        versionCode 7
+        versionName "1.2.0"
```

Comment:
maybe this doesn't matter much since it is an example code, but it is rare that the version name is increased and the version code is decreased 😅